prow/config.yaml

```diff
@@ -14278,3 +14278,40 @@ periodics:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+
+- name: periodic-test-infra-retester
+  interval: 20m  # Retest at most 1 PR per 20m, which should not DOS the queue.
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:latest
+      args:
+      - |-
+        --query=is:pr
+        -label:do-not-merge
+        label:lgtm
+        label:approved
+        -status:pending
+        -status:success
+        -label:needs-rebase
+        -label:needs-ok-to-test
+        -label:"cncf-cla: no"
+        repo:kubernetes/kubernetes
+      - --user=fejta-bot
+      - --token=/etc/token/bot-github-token
+      - |-
+        --comment=/retest
+        This bot automatically retries jobs that failed/flaked on approved PRs (send feedback to `@fejta`).
+
+        Review the [full test history](https://k8s-gubernator.appspot.com/pr/{{.Number}}) for this PR.
+
+      - --template
+      - --ceiling=1
+      - --wet
+      volumeMounts:
+      - name: token
+        mountPath: /etc/token
+    volumes:
+    - name: token
+      secret:
+        secretName: fejta-bot-token
```

Comment:
Yes because this bot has no rights to anything (it's equivalent to some internet rando leaving comments on PRs).

We do not want prow jobs to have write access to repos (this one does not).